### PR TITLE
Release 2.9.14-beta1

### PR DESCRIPTION
**Pull request by niik**
## Description
Looking for the PR for the upcoming beta of the v2.9.14 series? Well, you've just found it, congratulations!

This builds on 2.9.13 but contains dugite 1.106.0 which includes Git 2.35.2 and Git LFS 3.1.2

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
None
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
No Changes

### CODE REVIEW

**Review by tidy-dev — DISMISSED** [review]

File: app/test/setup-test-framework.ts [hunk 1]
```diff
@@ -3,3 +3,6 @@ jest.setTimeout(10000)
 
 import 'jest-extended'
 import 'jest-localstorage-mock'
+
+delete process.env['LOCAL_GIT_DIRECTORY']
```
Comment (tidy-dev) [inline app/test/setup-test-framework.ts:7]: Why is this needed? I figured tests must be breaking.. but just curious.

Comment (niik) [reply app/test/setup-test-framework.ts:7]: Yeah, see https://github.com/desktop/desktop/pull/14335/commits/2e80e8bc952366d654ead095381d6b0a15f4c6d0

> I had launched VS Code via Desktop and it had inherited these causing tests to use the prod version dugite

Comment (niik) [reply app/test/setup-test-framework.ts:7]: So this change isn't necessary on CI but it's gonna bite me again one of these days so I'd rather fix it now

**Review by tidy-dev — APPROVED** [review]
